observer/monitor.go

```diff
@@ -32,6 +32,7 @@ func (m monitor) start(logger blog.Logger) {
 			logger.Infof(
 				"kind=[%s] success=[%v] duration=[%f] name=[%s]",
 				m.prober.Kind(), success, dur.Seconds(), m.prober.Name())
-		}
-	}()
+		}()
+		<-ticker.C
```

Comment:
Another possibility here would be to factor out the inner function into a helper, call it directly between lines 20 and 21, and then keep the structure of the loop the same. Basically, run once, then do a normal `for range ticker.C` afterwards. 

```go
inner = func() {
  // all of lines 23 through 34
}
go inner()
for range ticker.C {
  go inner()
}
```
This is the approach that crl-updater takes, but it's doing considerably more complex logic around context cancellation at the same time, so we don't necessarily have to use the same approach here. But food for thought!